x-pack/plugins/cases/server/services/user_actions/types.ts

```diff
@@ -126,3 +134,24 @@ export type CommonBuilderArguments = CommonArguments & {
 export interface BuilderDeps {
   persistableStateAttachmentTypeRegistry: PersistableStateAttachmentTypeRegistry;
 }
+
+export interface ServiceContext {
```

Comment:
Refactored the parameters to the user action service into this context interface